src/freenet/node/NodeStarter.java

```diff
@@ -148,6 +144,11 @@ public Integer start(String[] args) {
 			return Integer.valueOf(-2);
 		}
 
+		String builtWithMessage = "freenet.jar built with freenet-ext.jar Build #" + ExtVersion.buildNumber + " r" + ExtVersion.cvsRevision+" running with ext build "+extBuildNumber+" r" + extRevisionNumber;
```

Comment:
This seems unrelated to the PR.
What's the reason for moving this around?

Reply:
It appeared before logging is started, and I don't find this message in the log file, so I moved it to after logging started.